repos/system_upgrade/el8toel9/actors/checkifcfg/libraries/ifcfg.py

```diff
@@ -0,0 +1,145 @@
+import os
+
+from leapp import reporting
+
+
+def process():
+    TRUE_VALUES = ['yes', 'true', '1']
+    GOOD_TYPES = [
+        'ethernet', 'ctc', 'wireless', 'infiniband', 'bond',
+        'team', 'vlan', 'bridge'
+    ]
+
+    rule_files = []
+    bad_type_files = []
+    not_controlled_files = []
+
+    try:
+        files = os.scandir('/etc/sysconfig/network-scripts')
+    except OSError:
+        # The legacy network-scripts package is probably already
+        # gone without a trace. Good!
+        return
+
+    for f in files:
+        bad_type = False
+        got_type = None
+        nm_controlled = True
+
+        if not f.is_file():
+            continue
+
+        if f.name.startswith('rule-') or f.name.startswith('rule6-'):
+            rule_files.append(f.path)
+            continue
+
+        if not f.name.startswith('ifcfg-'):
+            continue
+
+        if f.name == 'ifcfg-lo':
```

Comment:
If this configures anything for lo like DNS options, should this inhibit the upgrade, too? Seems best to request to get this file migrated, too.

Reply:
NetworkManager just ignores ifcfg-lo altogether. If you put DNS options into there, then it has no effect.

Reply:
Yes, if NetworkManager ignores this file but network-scripts does not, then this means the file needs to be migrated/the update needs to be blocked, otherwise the behavior might change/things might get broken.